src/client/sandbox/shadow-ui.js

```diff
@@ -406,28 +445,16 @@ export default class ShadowUI extends SandboxBase {
     }
 
     static isShadowContainer (el) {
-        if (domUtils.isDomElement(el))
-            return domUtils.isBodyElement(el) || domUtils.isHeadElement(el);
-
-        return false;
+        return domUtils.isBodyElement(el) || domUtils.isHeadElement(el);
```

Comment:
You can also mark `body` and `head` elements in the same way.
It is more fast the current condition